DQMOffline/L1Trigger/src/L1TEGammaOffline.cc

```diff
@@ -180,6 +227,7 @@ void L1TEGammaOffline::fillElectrons(edm::Event const& e, const unsigned int nVe
       minDeltaR = currentDeltaR;
       closestL1EGamma = *egamma;
       foundMatch = true;
+      break;
```

Comment:
Why do you break after the first match is found? It might not be the best match. Imagine there are two EG candidates and the first one matches with a dR of 0.29, while the second one matchs with dR of 0.05. The previous code would use the second one as the matching L1 EG, while the new code uses the first EG candidate.

Reply:
ah, my bad, I accidentally put this in while still in the L1Jet mindset